### PR TITLE
VMO-4307 + VMO-4308 + VMO-4309  Fixes for these tickets

### PR DESCRIPTION
**Pull request by poojahpatil90**
VMO-4307 

Screen Shot 2021-07-20 at 2 18 36 PM

VMO-4308 
Screen Shot 2021-07-20 at 2 19 52 PM

VMO-4309
Screen Shot 2021-07-20 at 2 20 31 PM

### CODE REVIEW

**Comment by poojahpatil90** [comment]
@asalami For Design review

**Comment by asalami** [comment]
@poojahpatil90 looks good to me!

File: src/components/interaction-designer/Connection.vue [hunk 1]
```diff
@@ -59,11 +59,11 @@ export default {
         // path: 'arc',
         // path: 'magnet',
 
-        middleLabel: LeaderLine.captionLabel(this.exit.tag, {
-          color: darkColor,
-          fontSize: 12,
-          // lineOffset: 65,
-        }),
+        // middleLabel: LeaderLine.captionLabel(this.exit.tag, {
```
Comment (safydy-r) [inline src/components/interaction-designer/Connection.vue:62]: I agree to leave this as comment

**Review by safydy-r — APPROVED** [review]
No comments, thanks

**Pull request merged**